cmd/commands/runtime.go

```diff
@@ -1727,6 +1732,17 @@ func inferProviderFromCloneURL(cloneURL string) (apmodel.GitProviders, error) {
 	return apmodel.GitProviders(""), fmt.Errorf("failed to infer git provider from clone url: %s, %s", cloneURL, suggest)
 }
 
+func inferProviderFromInsProvider(provider string) apmodel.GitProviders {
+	if provider == "github" {
```

Comment:
you can get rid of this function and do `apmodel.GitProviders(strings.ToUpper(provider))`